src/feature_annotation/score_genes_cell_cycle_scanpy/config.vsh.yaml

```diff
@@ -0,0 +1,81 @@
+functionality:
+  name: score_genes_cell_cycle_scanpy
+  namespace: feature_annotation
+  description: |
+    Calculates the score associated to S phase and G2M phase and annotates the cell cycle phase for each cell, as implemented by scanpy. 
+    The score is the average expression of a set of genes subtracted with the average expression of a reference set of genes.
+  argument_groups:
+
+    - name: Inputs
+      arguments:
+        - name: --input
+          type: file
+          description: Input h5mu file
+          required: true
+          example: input_file.h5mu
+        - name: --s_genes
+          type: file
+          description: |
+            Path to a .txt file containing the gene list of s phase genes to be scored. 
+            The gene list file should be formatted as a single column with gene symbols.
+          required: true
+          example: gene_list.txt
+        - name: --g2m_genes
+          type: file
+          description: |
+            Path to a .txt file containing the gene list of g2m phase genes to be scored. 
+            The gene list file should be formatted as a single column with gene symbols.
+          required: true
+          example: gene_list.txt
+        - name: --gene_pool
+          type: file
+          description: |
+            File with genes for sampling the reference set. Default is all genes. 
+            The gene pool file should be formatted as a single column with gene symbols.
+          required: false
+          example: gene_pool.txt
+
```

Comment:
Most commonly, scanpy uses `.var_names` to determine the gene names. However, gene names are not always uniquely mapped to ENSEMBL id for example. What we most commonly do is put the ENSEMBL id at `.var_names` and the gene names in a column in var and the name of the gene symbol column is up to the user. Could you add a `var_gene_names` argument to select the column that is used? This means that you need to add this column as `var_names` to a custom AnnData object that you pass to scanpy.